backend/src/main/java/de/nordakademie/iaa/noodle/services/implementation/MailServiceImpl.java

```diff
@@ -147,6 +156,41 @@ public void sendNeedsAttentionMail(Survey survey, User participant) throws MailC
         sendMail("Survey update", body, participant.getEmail());
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void sendSurveyClosedMailsAsync(Survey survey) {
+        new Thread(() ->
+            survey.getParticipations()
+                .stream()
+                .filter(participation -> participation.getResponse() != null)
+                .filter(participation -> !participation.getParticipant().equals(survey.getCreator()))
+                .map(Participation::getParticipant)
+                .forEach(user -> {
+                    try {
+                        sendNeedsAttentionMail(survey, user);
+                    } catch (MailClientException e) {
+                        System.err.println("Failed to send needs attention mail for " + user.getEmail());
+                    }
+                })).start();
+    }
```

Comment:
Can we split this up a bit? That lambda could just be named like the the function without "async" and we could wrap it in a "send async" Method or something. I think it's a bit too rich for a lambda.

Reply:
Sure.